src/sent_name_server.cpp

```diff
@@ -0,0 +1,55 @@
+//
+// Created by hlf on 5/10/23.
+//
+// !server.cpp
+#include "ros/ros.h"
+#include "std_msgs/String.h"
+#include "helifeng/SentName.h"
+
+void chatterCallback(const std_msgs::String::ConstPtr &msg) {
+    ROS_INFO("name: [%s]", msg->data.c_str());
+}
+
+bool show(helifeng::SentName::Request &req,
+          helifeng::SentName::Response &res) {
+    ROS_INFO("name: %s", (std::string) req.name);
+    //ros::init(argc, argv, "listener");
+    //ros::NodeHandle listener_nh;
+    //ros::Subscriber sub = listener_nh.subscribe("chatter", 100, chatterCallback);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]